barna-ajedrez/src/components/RegisterClubComponents/RegisterClubForm.js

```diff
@@ -0,0 +1,248 @@
+      { value: 'Chino', index: 8 },
+      { value: 'Ruso', index: 9 }
+    ];
+    this.clubExtras = [
+      { value: 'Bar', index: 1 },
+      { value: 'Clases', index: 2 },
+      { value: 'Wifi', index: 3 },
+      { value: 'Reuniones', index: 4 },
+      { value: 'Dominos', index: 5 },
+      { value: 'Internacional', index: 6 },
+      { value: 'Cerca del Metro', index: 7 },
+      { value: 'Jugadores Titelado', index: 8 },
+      { value: 'Amigable para principiantes', index: 9 }
+    ];
+  }
+  handleSubmit = e => {
+    e.preventDefault();
+    // checks are working before sending to back end
+    // const pChecker = (this.state.password === this.state.password2) ? true : false;
+    // const eChecker = (this.state.clubEmail === this.state.clubEmail2) ? true : false;
+    // const nChecker = (this.state.clubNumber === this.state.clubNumber2) ? true : false;
+    // console.log(pChecker);
+
+    // if (!pChecker) alert('please make sure that your passwords match');
+    // if (!eChecker) alert('please make sure that your emails are the same');
+    // if (!nChecker) alert('please make sure that the phone numbers are the same');
+    const value = this.state;
+
+    console.log(value);
+
+    axios
+      .post('http://localhost:3001/CreateNewClub', {
+        clubName: value.clubName,
+        location: value.location,
+        organizerName: value.organizerName,
+        clubNumber: value.clubNumber,
+        clubEmail: value.clubEmail,
+        password: value.password,
+        languages: [],
+        extras: [],
+        clubWebsite: value.clubWebsite,
+        additionalInfo: value.additionalInfo
+      })
+      .then(res => {
+        console.log(res);
+        alert('Account Created Successfully');
+      })
+      .catch(err => {
+        // properly receives
+        console.log('this is the error', err);
+        alert('Email exists');
+      });
+  };
+
+  handleChange = e => {
+    let value = e.target.value;
+    let name = e.target.name;
+    let nChecker = name === 'languages' || name === 'extras' ? true : false;
+
+    if (nChecker) {
+      this.setState(prevState => ({
+        [name]: [...prevState[name], value]
+      }));
+    } else {
+      this.setState({ [name]: value });
+    }
+  };
+
+  render() {
+    return (
+      <div className="rc-container">
+      <h2>Register your club</h2>
+        <form className="rc-form">
+          Nombre del Club:{' '}
+          <input
+            className="f-input"
+            name="clubName"
+            type="text"
+            onChange={this.handleChange}
+            value={this.state.value}
+            clubname="clubName"
+          />
+          Dirección:{' '}
+          <input
+            className="f-input"
+            name="location"
+            type="text"
+            onChange={this.handleChange}
+            value={this.state.value}
+            location="location"
+          />
+          Organizador(a):{' '}
+          <input
```

Comment:
All these inputs have similar structure, you can create a presentational component in this same file.